src/Illuminate/Auth/Passwords/PasswordBroker.php

```diff
@@ -55,6 +55,12 @@ public function sendResetLink(array $credentials)
             return static::INVALID_USER;
         }
 
+        // An attacker can make a lot of password reset requests,
+        // which will lead to spam in user's mailbox.
+        if ($this->tokens->recentlyCreated($user)) {
```

Comment:
BC here, this method is not part of the interface.

Possible solutions:
- Create an interface for it like ThrottleableTokenRepository and checks against that before calling this.
- Include it in the interface in master and check with `method_exists` in 6.x